### PR TITLE
Supporting github.ref variable for branch input

### PR DESCRIPTION
**Pull request by isaacadams**
#54 before using the branch input, it will check for the github.ref formatting and reformat accordingly

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ async function main() {
         }
 
         if (!runID) {
+            branch = branch.startsWith('refs/heads/') ? branch.split('refs/heads/')[1] : branch;
```
Comment (dawidd6) [inline main.js:61]:
```suggestion
            branch = branch.replace(/^refs\/heads\//, "")
```

Comment (dawidd6) [reply main.js:61]: Also please move it up a bit.

**Comment by dawidd6** [comment]
Thanks, gonna tweak it a bit on master. Merging.

**Comment by isaacadams** [comment]
@dawidd6 tweak away! thanks for accepting the PR :tada:

**Pull request merged**